apis/quay/v1/quayregistry_types.go

```diff
@@ -752,6 +754,36 @@ func GetEnvOverrideForComponent(quay *QuayRegistry, kind ComponentKind) []corev1
 	return nil
 }
 
+func GetLabelsOverrideForComponent(quay *QuayRegistry, kind ComponentKind) map[string]string {
```

Comment:
Haven't checked but I believe this is generating a warning on my lint here. We should always add comments to "exported" functions (the ones that start with an uppercase). I would suggest you to read the [Effective Go](https://go.dev/doc/effective_go), for this specific case the section "Commentary".